Library/Homebrew/utils/inflection.rb

```diff
@@ -0,0 +1,18 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Utils
+  # Inflection utility methods, as a lightweight alternative to `ActiveSupport::Inflector``.
+  #
+  # @api private
+  module Inflection
```

Comment:
I know this matches ActiveSupport but similarly perhaps making this just `Utils.pluralize` may be a nicer fit with the existing API.

Reply:
Can do, though it feels materially different from the others at the top level:
```
irb(main):001:0> Utils.methods(false)
=> [:safe_popen_read, :git_commit_message, :popen, :git_short_head, :git_branch, :popen_read, :popen_write, :safe_popen_write, :git_head, :rewrite_child_error, :safe_fork]
```

Reply:
@dduugg Honestly I think those ones have just ended up there because we didn't use to separate into submodules. I think the benefit of having this just be `Utils::pluralize` or, probably overkill, even in or wrapped in `utils.rb` as `pluralize` is it makes invocations, usually interpolated, much shorter.